source/gui/__init__.py

```diff
@@ -582,27 +596,7 @@ def wx_CallAfter_wrapper(func, *args, **kwargs):
 	wx.CallAfter = wx_CallAfter_wrapper
 
 def terminate():
-	import brailleViewer
-	brailleViewer.destroyBrailleViewer()
-
-	for instance, state in gui.SettingsDialog._instances.items():
-		if state is gui.SettingsDialog._DIALOG_DESTROYED_STATE:
-			log.error(
-				"Destroyed but not deleted instance of settings dialog exists: {!r}".format(instance)
-			)
-		else:
-			log.debug("Exiting NVDA with an open settings dialog: {!r}".format(instance))

```

Comment:
Please keep this code. Happy for it to be moved to another function with a comment from:
https://github.com/nvaccess/nvda/pull/12291#pullrequestreview-636204844

Reply:
I've kept and moved the changed code from #12291 to `safeAppExit`.